tests/Sulu/Component/Rest/ListBuilder/Metadata/General/Driver/XmlDriverTest.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Metadata\General\Driver;
+
+use Metadata\ClassMetadata;
+use Metadata\Driver\FileLocatorInterface;
+use Prophecy\Argument;
+use Sulu\Component\Rest\ListBuilder\Metadata\General\PropertyMetadata;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+class XmlDriverTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var FileLocatorInterface
+     */
+    private $locator;
+
+    /**
+     * @var ParameterBagInterface
+     */
+    private $parameterBag;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->locator = $this->prophesize(FileLocatorInterface::class);
+        $this->parameterBag = $this->prophesize(ParameterBagInterface::class);
+    }
+
+    protected function loadMetadataFromFile(XmlDriver $driver, $file)
```

Comment:
There is a good convention that there should be a "visibility" ordering of methods: `public`, `protected` and then `private`. So I would move this method to the bottom.